app/file_sender.py

```diff
@@ -150,21 +153,58 @@ def start_file_sender(readiness_queue):
     with sftp.SftpUtility(Config.SFTP_PPO_DIRECTORY):
         logger.info('Successfully connected to SFTP PPD directory', sftp_directory=Config.SFTP_PPO_DIRECTORY)
     readiness_queue.put(True)
+
+    check_gcp_bucket_ready()
```

Comment:
The check should be done before signalling ready on the queue, that's used to indicate that the service has finished starting up and is now ready/processing messages/files